examples/ota-provider-app/esp32/main/BdxOtaSender.cpp

```diff
@@ -202,15 +213,19 @@ void BdxOtaSender::Reset()
 {
     mFabricIndex.ClearValue();
     mNodeId.ClearValue();
-    mTransfer.Reset();
-    if (mExchangeCtx != nullptr)
+
+    // mExchangeCtx->Close() will crash if ExchangeMgr is nullptr. So, check for nullptr before calling Close().
```

Comment:
How did we get to a state where the ExchangeMgr is null but the ExchangeCtx is not?

Reply:
@shubhamdp ?

Reply:
@woody-apple this is being worked on, this is the similar issue being fixed in #16615